tracks/python/exercises/raindrops/mentoring.md

```diff
@@ -31,30 +31,23 @@ def raindrops(val: int) -> str:
 Linear testing
 
 ```python
-def raindrops(number):
-    factors = ''
+def raindrops(number) 
+  
+    result = ''
     if number % 3 == 0:
-      factors = factors + 'Pling'
+      result = result + 'Pling'
     if number % 5 == 0:
-      factors = factors + 'Plang'
+      result = result + 'Plang'
```

Comment:
```suggestion
      result += 'Plang'
```

Reply:
@jeffdparker Just click "Commmit suggestion" above like the other two, and we'll be done here.